xbmc/cores/DllLoader/Win32DllLoader.cpp

```diff
@@ -324,6 +324,10 @@ void Win32DllLoader::OverrideImports(const std::string &dll)
 
 bool Win32DllLoader::NeedsHooking(const char *dllName)
 {
+  if (!(StringUtils::EndsWithNoCase(dllName, "libdvdcss-2.dll")
+     || StringUtils::EndsWithNoCase(dllName, "libdvdnav.dll"))
```

Comment:
Nope, does the right thing, though not very readable :-) the ! counts for
the second case, too, e.g. !(x || y) == !x && !y so basically what he wants
here.

2016-04-15 8:51 GMT+02:00 Anton Fedchin <notifications@github.com>:

> In xbmc/cores/DllLoader/Win32DllLoader.cpp
> <https://github.com/xbmc/xbmc/pull/9619#discussion_r59833217>:
>
> > @@ -324,6 +324,10 @@ void Win32DllLoader::OverrideImports(const std::string &dll)
> >
> >  bool Win32DllLoader::NeedsHooking(const char *dllName)
> >  {
> > +  if (!(StringUtils::EndsWithNoCase(dllName, "libdvdcss-2.dll")
> > +     || StringUtils::EndsWithNoCase(dllName, "libdvdnav.dll"))
>
> seems closing bracket was missed
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/xbmc/xbmc/pull/9619/files/4b03b6d86b4b87b47e917340110d1929fdb5ae7a#r59833217>
>



-- 
                   Key-ID:     0x1A995A9B
                   keyserver: pgp.mit.edu
==============================================================
Fingerprint: 4606 DA19 EC2E 9A0B 0157  C81B DA07 CF63 1A99 5A9B